### PR TITLE
nuget 5.4.0 release notes

### PR DESCRIPTION
**Pull request by rrelyea**
Automated draft of release notes.
@rrelyea tweaked the titles and metadata...

Please check to see if any work you did, that shipped in 5.4, is covered here...and the title is appropirate.

@karann-msft  - VS version: 16.4. NuGet Version: 5.4. DotNet SDK version: 3.1.100

If we tweak the issue titles manually in this md file, we should also modify the real issue title.

### CODE REVIEW

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[2040751](https://github.com/NuGet/docs.microsoft.com-nuget/commits/2040751973b2fa74105c9d12ced003325b53c64b)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/release-notes/NuGet-5.4.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-rrelyea-releaseNotesFor5.4/docs/release-notes/NuGet-5.4.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/nuget/release-notes/nuget-5.4?branch=pr-en-us-1799) | [Details](#user-content-0286fe1f-e3d005e4)

<a id="0286fe1f-e3d005e4"></a>
### [docs/release-notes/NuGet-5.4.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-rrelyea-releaseNotesFor5.4/docs/release-notes/NuGet-5.4.md)
 - **[Warning]** `````Missing attribute: title. Add a title string to show in search engine results.`````
 - **Line 1**: **[Warning]** `````A space is required after the hash (#) in H1.`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C12%5C4%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201912040105000059-1799%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=ej%2BGqN337AhuiKerOvhcZ%2FyBHrfOKQ6BV%2FriX7WbVgo%3D&st=2019-12-04T01%3A01%3A48Z&se=2020-01-04T01%3A06%3A48Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/release-notes/NuGet-5.4.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+#5.4 Release Notes
+
+[Full Changelog]()
```
Comment (zkat) [inline docs/release-notes/NuGet-5.4.md]: Is this supposed to be missing a link?

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[1b43403](https://github.com/NuGet/docs.microsoft.com-nuget/commits/1b434038f921690e5bb31c7717cb69d2bf10d595)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/release-notes/NuGet-5.4.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-rrelyea-releaseNotesFor5.4/docs/release-notes/NuGet-5.4.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/release-notes/nuget-5.4?branch=pr-en-us-1799) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C12%5C4%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201912041943136257-1799%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=E%2FRFBlJksciVdByy%2BzbwUMq2tyqbQ9x3BrT6hYd7I5I%3D&st=2019-12-04T19%3A39%3A22Z&se=2020-01-04T19%3A44%3A22Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[f30c853](https://github.com/NuGet/docs.microsoft.com-nuget/commits/f30c853d72c8e103fe8dacd879e1636ff29b319a)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/release-notes/NuGet-5.4.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-rrelyea-releaseNotesFor5.4/docs/release-notes/NuGet-5.4.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/release-notes/nuget-5.4?branch=pr-en-us-1799) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C12%5C4%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201912042027473713-1799%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=9q2Gg6YQNhH5zVLYrWvBQ9C4fMyk9Wl2qO10rmMqTk8%3D&st=2019-12-04T20%3A23%3A53Z&se=2020-01-04T20%3A28%3A53Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[4df2251](https://github.com/NuGet/docs.microsoft.com-nuget/commits/4df2251f580b87a260cf8601198c1a00f448b3bd)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/release-notes/NuGet-5.4.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-rrelyea-releaseNotesFor5.4/docs/release-notes/NuGet-5.4.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/release-notes/nuget-5.4?branch=pr-en-us-1799) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C12%5C4%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201912042113388332-1799%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=bdLqXNL6jSL0blIaVuwYpN6NgGnbLl4mjAkE6Pjg5j4%3D&st=2019-12-04T21%3A09%3A42Z&se=2020-01-04T21%3A14%3A42Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[290cb71](https://github.com/NuGet/docs.microsoft.com-nuget/commits/290cb71694bf2275cd85313983091cd29369fcb0)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/release-notes/NuGet-5.4.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-rrelyea-releaseNotesFor5.4/docs/release-notes/NuGet-5.4.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/release-notes/nuget-5.4?branch=pr-en-us-1799) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C12%5C4%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201912042123593454-1799%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=NK8SKtquTJ7OfzLdwQA0bu0LeCQO%2BqVl9HYN6MvkTXY%3D&st=2019-12-04T21%3A20%3A04Z&se=2020-01-04T21%3A25%3A04Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[e152a6b](https://github.com/NuGet/docs.microsoft.com-nuget/commits/e152a6b511883a099aecd2980e4f9076db89b6eb)_: 

### :x: Validation status: errors

Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master#build--publish) which may help to resolve issue.


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[]() | :x:Error | | [Details](#user-content-c1f71bab)

<a id="c1f71bab"></a>
###
 - **[Error]** `````Cannot sync git repo to specified state. Please make sure the pull request is mergeable. Details: Git merge results in conflict, repository remote url: https://github.com/NuGet/docs.microsoft.com-nuget, working directory: C:\LocalRun\W\05lb-s, target branch: master, target branch commit: 08819bcc4e4579cf16fee126b338082eef269278, source commit: e152a6b511883a099aecd2980e4f9076db89b6eb.`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C12%5C4%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201912042127553688-1799%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=6JsVOdgHGdHpxHKHETUOuObo%2F7Zh1FuJbJdLhTf6Xyc%3D&st=2019-12-04T21%3A23%3A08Z&se=2020-01-04T21%3A28%3A08Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[18adec5](https://github.com/NuGet/docs.microsoft.com-nuget/commits/18adec59ef3a7ed10797dd409f747c1795452a13)_: 

### :x: Validation status: errors

Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master#build--publish) which may help to resolve issue.


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[]() | :x:Error | | [Details](#user-content-f03d8f25)

<a id="f03d8f25"></a>
###
 - **[Error]** `````Cannot sync git repo to specified state. Please make sure the pull request is mergeable. Details: Git merge results in conflict, repository remote url: https://github.com/NuGet/docs.microsoft.com-nuget, working directory: C:\LocalRun\W\yg3j-s, target branch: master, target branch commit: 08819bcc4e4579cf16fee126b338082eef269278, source commit: 18adec59ef3a7ed10797dd409f747c1795452a13.`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C12%5C4%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201912042129336484-1799%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=yjXQy7MkNUwQyR4LoIGzgOpnEizTpby4eazs%2FDWmeUQ%3D&st=2019-12-04T21%3A24%3A45Z&se=2020-01-04T21%3A29%3A45Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[f469093](https://github.com/NuGet/docs.microsoft.com-nuget/commits/f469093b44b3a823ca46c90cd9dc4f02860497dd)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/TOC.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-rrelyea-releaseNotesFor5.4/docs/TOC.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/release-notes/index?branch=pr-en-us-1799) |
[docs/release-notes/Index.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-rrelyea-releaseNotesFor5.4/docs/release-notes/Index.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/release-notes/index?branch=pr-en-us-1799) |
[docs/release-notes/NuGet-5.4.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-rrelyea-releaseNotesFor5.4/docs/release-notes/NuGet-5.4.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/release-notes/nuget-5.4?branch=pr-en-us-1799) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C12%5C4%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201912042143241083-1799%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=wkQS1CELsqWQSF41L6wQ0vtmEFIJzWTH0bHKsPcUGuM%3D&st=2019-12-04T21%3A39%3A37Z&se=2020-01-04T21%3A44%3A37Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**